Week-3/Exercise-3.1/index.js

```diff
@@ -0,0 +1,30 @@
+function memorize(func) {
+    var cache = new Map();
+    return function (...args) {
+        var key = args.toString();
+        if(cache.has(key)){
+            result = cache.get(key);
+            // console.log('From cache: ', result);
+            return result;
+        } else {
+            cache.set(key, func(...args));
+            result = cache.get(key);
```

Comment:
where did u declare result

Reply:
I declared result for testing purpose only but forgot to remove before commit.
Update: I have removed that line of code because it was needed.